packages/table/test/tableTests.tsx

```diff
@@ -56,6 +56,24 @@ describe("<Table>", () => {
         expect(table.find(".bp-table-column-headers .bp-table-header", 1).element).to.be.ok;
     });
 
+    it("Renders correctly with loading options", () => {
+        const renderCell = () => <Cell>my cell value</Cell>;
+        const loadingOptions = [
+            TableLoadingOption.CELLS,
+            TableLoadingOption.COLUMN_HEADERS,
+            TableLoadingOption.ROW_HEADERS,
+        ];
+        const tableHarness = harness.mount(
+            <Table loadingOptions={loadingOptions} numRows={2}>
+                <Column name="Column0" renderCell={renderCell} />
+                <Column name="Column1" renderCell={renderCell} />
+            </Table>,
+        );
+        const tableElement = tableHarness.element.children[0];
+
+        expect(tableElement.textContent).to.equal("");
```

Comment:
the cells and column headers and row headers all should have text content. this is testing that all three of the table loading options are working correctly and causing all headers and cells to render loading state instead of content.